content/contributing/documentation/rst_cheat_sheet.rst

```diff
@@ -14,127 +14,309 @@ Headings
   encounter different heading formatting and in a different order, in which case you should follow
   the formatting in place in the document. In any other case, use the formatting shown below.
 
-+--------------+---------------+
-| Heading size | Formatting    |
-+==============+===============+
-| H1           | | ``=======`` |
-|              | | ``Heading`` |
-|              | | ``=======`` |
-+--------------+---------------+
-| H2           | | ``Heading`` |
-|              | | ``=======`` |
-+--------------+---------------+
-| H3           | | ``Heading`` |
-|              | | ``-------`` |
-+--------------+---------------+
-| H4           | | ``Heading`` |
-|              | | ``~~~~~~~`` |
-+--------------+---------------+
-| H5           | | ``Heading`` |
-|              | | ``*******`` |
-+--------------+---------------+
-| H6           | | ``Heading`` |
-|              | | ``^^^^^^^`` |
-+--------------+---------------+
++--------------+----------------------+
+| Heading size | Formatting           |
++==============+======================+
+| H1           | .. code-block:: text |
+|              |                      |
+|              |    =======           |
+|              |    Heading           |
+|              |    =======           |
++--------------+----------------------+
+| H2           | .. code-block:: text |
+|              |                      |
+|              |    Heading           |
+|              |    =======           |
++--------------+----------------------+
+| H3           | .. code-block:: text |
+|              |                      |
+|              |    Heading           |
+|              |    -------           |
++--------------+----------------------+
+| H4           | .. code-block:: text |
+|              |                      |
+|              |    Heading           |
+|              |    ~~~~~~~           |
++--------------+----------------------+
+| H5           | .. code-block:: text |
+|              |                      |
+|              |    Heading           |
+|              |    *******           |
++--------------+----------------------+
+| H6           | .. code-block:: text |
+|              |                      |
+|              |    Heading           |
+|              |    ^^^^^^^           |
++--------------+----------------------+
 
 .. important::
    Each document must have **exactly one H1 heading**. No less, no more.
 
-.. _contributing/markup:
+.. _contributing/markups:
 
-Markup
-======
+Markups
+=======
+
+.. _contributing/markups/italic:
+
+Emphasis (italic)
+-----------------
+
+To emphasize a part of the text. The text is rendered in italic.
+
+.. list-table::
+   :class: o-showcase-table
+
+   * - Fill out the information *before* saving the form.
+
+   * - .. code-block:: text
+
+          Fill out the information *before* saving the form. COIN COIN COINCOINCOINCOINCOINCOIN COINCOINCOINCOINCOINCOINCOINCOINCOINCOIN COINCOINCOINCOIN COIN COIN
```

Comment:
WTF dude... Be professional and use Lorem ipsum. 🤣
@AntoineVDV

Reply:
No one would have known if it wasn't for your comment!